### DISCUSSION TITLE
Welcome to bitsandbytes Discussions!

### DISCUSSION THREAD

**Discussion by TimDettmers** [bitsandbytes-foundation/bitsandbytes#989] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by philipbutler** [comment]
What's up! Poking around BNB today. Right now I'm trying to match torch's `torch.float8_e4m3fn` AdamW in CUDA, I suppose mainly for educational purposes. I currently think `torch.float8_e4m3fn` uses BNB? idk, lookin into it
